_stories/molecules/Breadcrumbs/index.js

```diff
@@ -66,39 +123,45 @@ class BreadcrumbsStory extends Component {
     static displayName = 'Breadcrumbs';
 
     state = {
-        hideSubmenus: false
+        hideSubmenus: false,
+        config: configA
+    };
+
+    changeConfig = () => {
+        const { config } = this.state;
+        const newConfig = config.path === 'home' ? configB : configA;
+        this.setState({ config: newConfig });
     };
 
+    printCode = code => '\n' + JSON.stringify(code, null, 4) + '\n\n';
+
     render() {
+        const { config } = this.state;
+        const configName = `Configuration ${config.path.includes('home') ? 'A' : 'B'}`;
         return (
-            <header className="gds-page-header">
-                <div className="gds-page-header__nav-bar">
-                    <Breadcrumbs
-                        config={config}
-                        pathname={select('pathname', routeOptions, routeOptions[0])}
-                        hideMenus={boolean('Hide Submenus', false)}
-                        hideRoot={boolean('Hide root breadcrumb', false)}
-                    />
-                </div>
-            </header>
+            <div>
+                <header className="gds-page-header">
+                    <div className="gds-page-header__nav-bar">
+                        <Breadcrumbs
+                            config={config}
```

Comment:
It's a little awkward but you can use the `select` knob to pick which config to use by doing this:

```js
const configSelectOptions = {
    configA: 'configA',
    configB: 'configB'
};

const configs = {
    configA,
    configB
};
```
and then use the knob like so:

```.jsx
<Breadcrumbs
    config={
        configs[
            select('Configuration', configSelectOptions, configSelectOptions['configA'])
        ]
    }
/>
```
I think we could also update path options based on the config selection too, but I haven't tried it yet.

Reply:
How would you pass the selection to the component to print it?

Reply:
https://gist.github.com/jscottsmith/3ea83a7fd3da3319e7c975ccc79bd32a

Reply:
Then to print the config just add 
```js
{this.printCode(configs[selectedConfig])}
```

Reply:
FYI, I cleaned up the gist a bit, and updated with the note from above.

Reply:
Done! Thanks for your help!